protostar/commands/cairo1_commands/declare_cairo1_command.py

```diff
@@ -96,16 +101,21 @@ async def run(self, args: Namespace) -> Any:
         )
 
         contract_name = args.contract
-        contract_sierra = (
-            self._project_compiler.compile_contract_to_sierra_from_contract_name(
-                contract_name
-            )
+        contract_path = self._contract_path_resolver.contract_path_from_contract_name(
+            contract_name
         )
-        contract_casm = (
-            self._project_compiler.compile_contract_to_casm_from_contract_name(
-                contract_name
+
+        contract_sierra = compile_starknet_contract_to_sierra_from_path(contract_path)
```

Comment:
prepending i.e. `cairo_1_bindings` would nicely show where this method comes from (clearer boundaries)

Reply:
This goes for all of the invocations in this PR